nautobot/extras/datasources/git.py

```diff
@@ -74,16 +74,11 @@ def enqueue_pull_git_repository_and_refresh_data(repository, user):
     return enqueue_git_repository_helper(repository, user, GitRepositorySync)
 
 
-def get_repo_from_url_to_path_and_from_branch(repository_record):
-    """Returns the from_url, to_path and from_branch of a Git Repo
+def get_repo_url(repository_record):
```

Comment:
I don't find this name super obvious. Maybe something like `get_authenticated_repo_url`?